### PR TITLE
Adding Hebrew language

### PR DESCRIPTION
**Pull request by eranroz**
adding Hebrew language module

### CODE REVIEW

File: revscoring/languages/tests/test_hebrew.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+    is_misspelled = hebrew.solve(language.is_misspelled)
+
+    assert is_misspelled("חטול")
+    assert not is_misspelled("חתול")
+
+    is_badword = hebrew.solve(language.is_badword)
+
+    assert is_badword("שרמוטה")
+    assert not is_badword("שימרותה")
```
Comment (halfak) [inline revscoring/languages/tests/test_hebrew.py]: Can you add a test for "is_informal_word"?

**Comment by halfak** [comment]
Looks like we just need one more test and this is good.  Thanks @eranroz!

File: revscoring/languages/tests/test_hebrew.py (hunk 1 above)
Comment (eranroz) [inline revscoring/languages/tests/test_hebrew.py]: Done

**Comment by halfak** [comment]
Thank you!

**Pull request merged**